app/routes/endemics/which-species-ms.js

```diff
@@ -12,6 +12,7 @@ const urlPrefix = require('../../config').urlPrefix
 const pageUrl = `${urlPrefix}/${endemicsWhichSpecies}`
 const backLink = claimDashboard
 const errorMessage = { text: 'Select which species you are claiming for' }
+const view = `${endemicsWhichSpecies}-ms`
```

Comment:
I dont particularly love the pattern they have where the name for all the views lives in the /config/routes folder, but as thats the pattern, do you think we should make a new one and use it here?